glance/manifests/api.pp

```diff
@@ -239,6 +239,21 @@
     warning('The mysql_module parameter is deprecated. The latest 2.x mysql module will be used.')
   }
 
+  if ($auth_protocol == 'https') {
+    class {'moc_openstack::ssl::add_glance_cert':
+      require => File['/etc/glance'],
+    }
+    class {'moc_openstack::ssl::temp_glance_fix':
```

Comment:
Added the fix. Currently, I am not updating the glanceclient version from 0.17 to 0.18 or more since its requiring many other packages to be updated (required me to install gcc, python-devel and libffi which I don't want). Hence, I am keeping the current patch, we will remove it once we have all the updated packages.